docs/source/admin/log-messages.md

```diff
@@ -1,22 +1,20 @@
 # Interpreting common log messages
 
-When debugging errors and outages, looking at the logs emitted by
-JupyterHub is very helpful. This document tries to document some common
-log messages, and what they mean.
+In this subsection of the documentation, you will get to understand the meaning of common log messages and how to resolve them. When debugging errors and outages, it is very helpful to look at the logs emitted by JupyterHub.
 
 ## Failing suspected API request to not-running server
 
 ### Example
 
-Your logs might be littered with lines that might look slightly scary
+When your screen displays the log message as shown below, you need not to get scared.
```

Comment:
It's not about the screen, it's the logs. If we're keeping "scary", think it's better to keep it as a property of the logs, rather than assigning it to the reader.

Reply:
Resolved